layout/js/gages.js

```diff
@@ -67,8 +78,8 @@
     vizlab.showyear(year);
   };
   
-  window.vizlab = window.vizlab || {}; // remove this
   vizlab.showyear = function(year) {
```

Comment:
@jiwalker-usgs , showyear was not working so I'm returning immediately. Will see if I can figure out the issue